Assets/Scripts/UI/Canvas/PhotoTaking/CameraPanel.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine;
+using TMPro;
+
+using Framework.ScriptableObjects;
+using UnityEngine.Serialization;
+
+namespace UI.Canvas.PhotoTaking
+{
+    public sealed class CameraPanel : MonoBehaviour
+    {
+        [SerializeField] private TMP_Text pointOfInterestName;
+        [SerializeField] private Texture2D pointOfInterestRender;
+
+        /// <summary>
+        /// Fills the camera panel with data from the given scriptable object
+        /// </summary>
+        /// <param name="photoData"> The scriptable object used to get data </param>
+        public void SetPanelData(PhotoData photoData)
```

Comment:
```suggestion
        public void SetPanelData(PhotoData tragetPhotoData)
```
Is mogelijk, hoeft niet perse.